src/WebSocket/WebSocketDispatcher.ts

```diff
@@ -0,0 +1,114 @@
+/*
+*  Send and receive requests serially
+*  next reuqest is only sent when the previous one got its response
+*/
+
+import { Dispatcher } from '../api';
+
+const errorMessages = {
+  notConnected: 'Not connected!',
+  resourceIsBusy: 'Resource is busy!',
```

Comment:
do we need exclamations?

Reply:
Why not? It is just to emphasize the severity of the message!